### ISSUE TITLE
Let's document: SerenityOS

### ISSUE THREAD

**Issue by Yutyo** [tldr-pages/tldr#5500]
Its quite a neat OS born in Github.

**Comment by bl-ue** [comment]
Ha! Yes, I've been seeing it.  https://github.com/SerenityOS/serenity

I've even opened a discussion on it :) https://github.com/SerenityOS/serenity/discussions/5845

Is it a command/set of commands, thought, @Yutyo?  How are you proposing to document it?

**Comment by sbrl** [comment]
Oooh, nice! Looks like a cool OS there - and that's a solid set of core libraries implemented too! Not sure I'd use it as a daily driver or for anything serious, but it does look interesting. Documenting it in tldr-pages might be a good way for new users to quickly get up and running with it. I assume that at least 1 of our existing clients would run on it lol - the advantages of having a diverse client ecosystem!

Anyway, do they have man pages or an equivalent for platform-specific commands yet? We'd really need some form of help text to work from to write pages here. It might also be a good idea to draw up a short list of say the 15 most useful commands and document those first. As for how to determine the most 15 most useful commands, is there anywhere we could ask users of the OS for suggestions perhaps? Or some documentation we can look at?

Of course, if any commands are compatible with existing pages in `common` then there's no need to document them specifically for SerenityOS.

In terms of _where_ to document them, platform-specific commands would need a new platform folder. I'd suggest that `serenityos` might be a good candidate name here, but `serenity` might be more approriate if we want to drop the `os` bit. I'm on the fence on that though, as the project appears to use _SerenityOS_ as their official name. Thoughts?

**Comment by vladimyr** [comment]
tl;dr Way too soon my friend, Serenity isn't even at _kindergarten age_ yet

---

If someone told me about this issue I wouldn't have believed them. And rest assured that I have high regards for Serenity and the whole community led by Andreas. :upside_down_face: 

> Documenting it in tldr-pages might be a good way for new users to quickly get up and running with it. I assume that at least 1 of our existing clients would run on it lol - the advantages of having a diverse client ecosystem!

You officially made me laughing out loud. :rofl: Thing is, there is no Serenity iso, you aren't supposed to run it on bare metal (yet) so these _new users_ are pretty much exclusively new Serenity devs and I'd bet they can survive without tldr client (for now). FWIW Serenity is largely POSIX compliant and if one is able to port VIM as-is (without patching) https://github.com/SerenityOS/serenity/blob/d454926/Ports/vim/package.sh then, I believe, porting tldr C client is definitely a child's play. :wink: 

> Anyway, do they have man pages or an equivalent for platform-specific commands yet? We'd really need some form of help text to work from to write pages here. It might also be a good idea to draw up a short list of say the 15 most useful commands and document those first. As for how to determine the most 15 most useful commands, is there anywhere we could ask users of the OS for suggestions perhaps? Or some documentation we can look at?

OFC they do https://github.com/SerenityOS/serenity/tree/d454926/Base/usr/share/man, and they are fairly modern - written in markdown :tada: 
Regarding most used commands, again you might just ask Andreas and the rest of the crew to send you their shell history :grinning: 

> Of course, if any commands are compatible with existing pages in `common` then there's no need to document them specifically for SerenityOS.

They are somewhat compatible due to Serenity's practice of implementing as much as they currently need plus not having Linux compatibility as a project goal, i.e. your mileage may vary.

> In terms of _where_ to document them, platform-specific commands would need a new platform folder. I'd suggest that `serenityos` might be a good candidate name here, but `serenity` might be more approriate if we want to drop the `os` bit. I'm on the fence on that though, as the project appears to use _SerenityOS_ as their official name. Thoughts?

Needless to say, their PR department hasn't released a press kit yet so we'll have to use our instincts here. :grinning: 

**PS** I'm fighting myself really hard here to not /cc Andreas. Watching a tonne of his YT works I'd bet he'd have a really good laugh. :grinning:
